### DISCUSSION TITLE
Add links to documents

### DISCUSSION THREAD

**Discussion by drichardson** [tosdr/edit.tosdr.org#1011] [Ideas]
I've seen some documents in the system that do not have links, which means there is no button to re-crawl the site. In such cases, I've added another document with a duplicate name (i.e. Privacy Policy) that is linked to the TOS.

This is an enhancement request to be able to update a document to point to the document URL if there isn't one already or if the link is broken.

**Comment by drichardson** [comment]
Here's an example of a document with no link:

https://edit.tosdr.org/documents/336

And here's an example of the new document I made to replace it with a duplicate name:

https://edit.tosdr.org/documents/1938

**Comment by michielbdejong** [comment]
You're absolutely right! The problem is we have an author+reviewer system for points, but for all other entities (cases, topics, services, documents) we have an author+commenter system. And so apart from the person who created a document, others can only comment. Unless you have moderator status, then I think you can also edit documents that you did not create.

I made you a curator now, so maybe that helps!

Thanks for your work and contributions!

**Comment by drichardson** [comment]
Ah. What happens when you re-crawl a document and the content changes. Does it invalidate existing annotations on that document?

**Comment by Vinnl** [comment]
@drichardson If the exact annotation is still present in the new Document, it will remain valid and will be linked to its new location. If not, it will be marked as "Pending" again.
